README.md

```diff
@@ -2,142 +2,155 @@
+  - [2. SSE to stdio](#2-sse-to-stdio)
+    - [2.1 Configuration](#21-configuration)
+    - [2.2 Example usage](#22-example-usage)
   - [Installation](#installation)
-  - [Configuration](#configuration)
-    - [Claude Desktop Configuration](#claude-desktop-configuration)
-  - [Advanced Configuration](#advanced-configuration)
-    - [Environment Variables](#environment-variables)
+  - [Testing](#testing)
 
 ## About
 
-Connect to MCP servers that run on SSE transport, or expose stdio servers as an SSE server using the MCP Proxy server.
+The `mcp-proxy` is a tool that lets you switch between server transports. There are two supported modes:
 
-## stdio to SSE
+1. stdio to SSE
+2. SSE to stdio
 
-```mermaid
-graph LR
-    A["Claude Desktop"] <--> B["mcp-proxy"]
-    B <--> C["External MCP Server"]
+## 1. stdio to SSE
 
-    style A fill:#ffe6f9,stroke:#333,color:black,stroke-width:2px
-    style B fill:#e6e6ff,stroke:#333,color:black,stroke-width:2px
-    style C fill:#e6ffe6,stroke:#333,color:black,stroke-width:2px
-```
-
-> [!TIP]
-> As of now, Claude Desktop does not support MCP servers that run on SSE transport. This server is a workaround to enable the support.
+Run a proxy server from stdio that connects to a remote SSE server.
 
-## SSE to stdio
+This mode allows clients like Claude Desktop to communicate to a remote server over SSE even though it is not supported natively.
 
 ```mermaid
 graph LR
-    A["LLM Client"] <--> B["mcp-proxy"]
-    B <--> C["Local MCP Server"]
+    A["Claude Desktop"] <--> |stdio| B["mcp-proxy"]
+    B <--> |SSE| C["External MCP Server"]
 
     style A fill:#ffe6f9,stroke:#333,color:black,stroke-width:2px
     style B fill:#e6e6ff,stroke:#333,color:black,stroke-width:2px
     style C fill:#e6ffe6,stroke:#333,color:black,stroke-width:2px
 ```
 
+### 1.1 Configuration
 
-## Installation
-
-The stable version of the package is available on the PyPI repository. You can install it using the following command:
-
-```bash
-uv tool install mcp-proxy
-```
-
-The latest version of the package can be installed from the git repository using the following command:
+This mode requires the URL of the SSE server to be set. The `API_ACCESS_TOKEN` environment variable can be set to pass the access token to the remote server.
 
-```bash
-uv tool install git+https://github.com/sparfenyuk/mcp-proxy
-```
-
-> [!NOTE]
-> If you have already installed the server, you can update it using `uv tool upgrade --reinstall` command.
-
-> [!NOTE]
-> If you want to delete the server, use the `uv tool uninstall mcp-proxy` command.
+Arguments
 
-## Configuration
+| Name           | Required                   | Description                               | Example               |
+| -------------- | -------------------------- | ----------------------------------------- | --------------------- |
+| command_or_url | Yes, if `SSE_URL` is unset | The MCP server SSE endpoint to connect to | http://example.io/sse |
```

Comment:
This is the first mention of `SSE_URL` and it is not mentioned elsewhere. Do you want to also add to the env var list?

Reply:
Yes, looks great. I was also tempted to make this same change, but didn't want to totally change everything without your permission.